### PR TITLE
Add documentation for Websocket API

### PR DESCRIPTION
**Pull request by shal**

### CODE REVIEW

File: docs/api/websocket_api.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+**Note:** Peatio websocket API supports authentication only with Bearer type of JWT token.
+
+**Example** of authentication message:
+
+```JSON
+{
+  "jwt": "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWV9.TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ"
+}
+```
```
Comment (mod) [inline docs/api/websocket_api.md:67]: We need the list of subscription channels and information between what message goes on public channel and which ones on private channel @yivo @ashanaakh

File: docs/api/websocket_api.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+| `side`          | Either 'sell' or 'buy'.                                 |
+| `ord_type`      | Type of order, either 'limit' or 'market'.              |
+| `price`         | Price for each unit.                                    |
+| `avg_price`     | Average execution price, average of price in trades.    |
+| `state`         | One of 'wait', 'done', or 'cancel'.                     |
+| `market_id`     | The market in which the order is placed, e.g. 'btcusd'. |
+| `created_at`    | Order create time in iso8601 format.                    |
+| `origin_volume` | The amount user wnt to sell/buy.                        |
+| `trades_count`  | Number of trades.                                       |
+| `trades`        | List of trades.                                         |
```
Comment (mod) [inline docs/api/websocket_api.md]: I believe there is other message types.

**Review by mod — CHANGES_REQUESTED** [review]

**Comment by shal** [comment]
@mod as I see in the websocket protocol source code, here is **no** private/public channels.
Here:
https://github.com/rubykube/peatio/blob/8537db2b28e35ff71b64a35d494207bd302e6739/app/api/api_v2/websocket_protocol.rb#L47-L48

and here:

https://github.com/rubykube/peatio/blob/8537db2b28e35ff71b64a35d494207bd302e6739/app/api/api_v2/websocket_protocol.rb#L61-L62

I think it is initialized only once on the websocket connection.
Here:
https://github.com/rubykube/peatio/blob/8537db2b28e35ff71b64a35d494207bd302e6739/lib/daemons/websocket_api.rb#L19

Also for me important to mention, that API has on three main types of messages:
- `orderbook`
- `trade`
- `order`

File: docs/api/websocket_api.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
+# Peatio WebSocket API
```
Comment (yivo) [inline docs/api/websocket_api.md:1]: @ashanaakh Can you add some small Ruby script into this doc to demonstrate usage?

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by mod** [comment]
I don't understand how to use this as API by reading this md file, use more English words please describe make a drawing, do whatever it takes

**Comment by shal** [comment]
@mod I think there is no use to add drawing here. @yivo I've added ruby code in the bottom, looks much clear for me.

**Review by yivo — APPROVED** [review]

**Pull request merged**